eclair-node/pom.xml

```diff
@@ -102,6 +120,32 @@
             <artifactId>akka-http-json4s_${scala.version.short}</artifactId>
             <version>1.19.0</version>
         </dependency>
+        <!-- metrics -->
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-apm-reporter_${scala.version.short}</artifactId>
+            <version>${kamon.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-system-metrics_${scala.version.short}</artifactId>
+            <version>${kamon.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-jdbc_${scala.version.short}</artifactId>
+            <version>${kamon.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-annotation_${scala.version.short}</artifactId>
+            <version>2.0.0-RC1</version>
+        </dependency>
+        <!--dependency>
```

Comment:
Maybe remove entirely? I don't expect it to be useful to monitor akka-http since we only use it for the RPC API...

Reply:
I think monitoring the API is still useful, and I'm waiting for a compatible version.